src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Http/ClientCredentialTypeTests.cs

```diff
@@ -26,7 +26,10 @@ public static void DigestAuthentication_Echo_RoundTrips_String_No_Domain()
             binding = new BasicHttpBinding(BasicHttpSecurityMode.TransportCredentialOnly);
             binding.Security.Transport.ClientCredentialType = HttpClientCredentialType.Digest;
             factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.Http_DigestAuth_NoDomain_Address));
-            factory.Credentials.HttpDigest.ClientCredential = BridgeClientAuthenticationManager.NetworkCredential;
+
+            // https://github.com/dotnet/wcf/issues/1045 needs to supply a replacement for NetworkCredential below
```

Comment:
Reviewers: this test is marked with [ActiveIssue(1045)] so was not executing.  Now we know why it failed -- it was trying to invoke Bridge logic.  Issue 1045 will be responsible for providing a new way to do this.